internal/api/integrations/k8s/k8s.go

```diff
@@ -76,16 +85,44 @@ func (k *K8sClient) ListAll(namespace string) ([]models.Release, error) {
 				Sumologic: annotations[annotationFor("sumologic")],
 			},
 			Code: models.SourceCode{
-				Github: annotations[annotationFor("code")],
+				Github: findVCSRepo(codeURL),
+				Ref:    image.Tag,
 			},
 			Artifacts: models.Artifacts{
 				Chart: models.HelmArtifact{
 					Path:    r.Spec.Chart.Path,
 					Version: r.Spec.Chart.Revision,
 				},
+				Docker: models.DockerArtifact{
+					Image: image.Repository,
+					Tag:   image.Tag,
+				},
 			},
 		})
 	}
 
 	return releases, nil
 }
+
+func findVCSRepo(url string) string {
+	arr := strings.Split(url, "/")
+	if len(arr) > 4 {
+		return arr[4]
+	}
+	return ""
+}
+
+func GetImageForRepo(repo string, vals map[string]interface{}, logger log.Logger) internal.Image {
+	arr := internal.GetImagePath(vals, repo)
+	if len(arr) == 0 {
+		logger.Log("No Image Found for repository")
+		return internal.Image{}
+	}
+	imgVals := internal.Table(vals, arr)
+	img, err := internal.ParseImage(imgVals["repository"].(string), imgVals["tag"].(string))
+	if err != nil {
+		logger.Log("Unable to parse valid image")
```

Comment:
This message is confusing, we couldn't parse a _valid_ image? This log message should include the error message as well.